src/Twig/RolesMatrixExtension.php

```diff
@@ -71,14 +71,15 @@ public function renderMatrix(Environment $environment, FormView $form): string
     {
         $groupedRoles = [];
         foreach ($this->rolesBuilder->getRoles() as $role => $attributes) {
-            if (!isset($attributes['admin_label'])) {
+            if (!isset($attributes['admin_code'])) {
                 continue;
             }
 
-            $groupedRoles[$attributes['admin_label']][$role] = $attributes;
+            $groupCode = $attributes['group_code'] ?? '_';
```

Comment:
The `_` default seems weird.

Reply:
How to group admins without group? I can replace it to "default".

Reply:
Let's go with empty string. If we would have use `null` it would have been casted to `''` when used as an array key so, that's a good representation I would think.